lib/db/mysql.js

```diff
@@ -1189,6 +1189,14 @@ module.exports = function (log, error) {
     return this.readFirstResult(CONSUME_SIGNIN_CODE, [ createHash(code), newerThan ])
   }
 
+  const ACCOUNT_RESET_TOKENS = 'CALL resetAccountTokens_1(?)'
```

Comment:
Not sure how important they are but should this have one of the boilerplate `// Delete : ... Where : ...` comments like the other functions in here?